test/firtool/register-randomization.fir

```diff
@@ -50,11 +47,11 @@ circuit Foo:
     ; NONE:   automatic logic [31:0] _RANDOM_0;
     ; NONE:   automatic logic [31:0] _RANDOM_1;
     ; NONE:   automatic logic [31:0] _RANDOM_2;
-    ; NONE:   automatic logic [31:0] _RANDOM_3;
+    ; NONE-NOT:   automatic logic [31:0] _RANDOM_3;
```

Comment:
If these don't exist anymore, it may be fine to just remove these as opposed to add an explicit "not" check. I may be misunderstanding what's going on.

Reply:
Agreed. I removed "not" checks. Note that these tests changes are rather caused by https://github.com/llvm/circt/pull/3802. #3802 removed `automatic logic [31:0] _RANDOM_3;` in `Foo` module but `NONE:   automatic logic [31:0] _RANDOM_3;` matched to the register in `TwoLargeRegisters`